pkgs/applications/version-management/sr-ht/git.nix

```
@@ -0,0 +1,44 @@
+{ stdenv, fetchgit, fetchPypi, buildPythonApplication
+, srht, redis, pygit2
+, python }:
+
+let
+  redis_2 = redis.overrideAttrs (oldAttrs: rec {
```

```suggestion
  redis_2 = redis.overridePythonAttrs (oldAttrs: rec {
```
You should also use `python.override { packageOverrides = self: super: { ... }; }` and this overriding is probably best done in `default.nix`.